ee/tabby-webserver/src/db.rs

```diff
@@ -18,7 +19,83 @@ lazy_static! {
                 CONSTRAINT `idx_token` UNIQUE (`token`)
             );
         "#
-    ),]);
+        ),
+        M::up(
+            r#"
+            CREATE TABLE IF NOT EXISTS users (
+                id           INTEGER PRIMARY KEY AUTOINCREMENT,
+                username     VARCHAR(150) NOT NULL COLLATE NOCASE,
+                email        VARCHAR(150) NOT NULL COLLATE NOCASE,
+                password     VARCHAR(128) NOT NULL,
+                is_superuser BOOLEAN NOT NULL DEFAULT 0,
+                is_active    BOOLEAN NOT NULL DEFAULT 1,
```

Comment:
remove - this field can be added when we have a interface of `mutation disable user`